values/otomi-api/otomi-api.gotmpl

```diff
@@ -51,8 +51,8 @@ tools:
   enabled: {{ $v.sops.enabled }}
   {{- if $v.sops.enabled }}
   image:
-    pullPolicy: {{ $o | get "tools.image.pullPolicy" "IfNotPresent" }}
-    tag: {{ $o | get "tools.image.tag" "latest" }}
```

Comment:
I'm curious, why should we hardcode this? I can imagine that we will bump tools often? @Morriz

Reply:
Also, is it part of the task?

Reply:
resolved in another conversation ;)